src/DependencyInjection/Configuration.php

```diff
@@ -516,14 +516,15 @@ public function getConfigTreeBuilder()
 
                 ->arrayNode('extensions')
                 ->useAttributeAsKey('id')
-                ->defaultValue(['admins' => [], 'excludes' => [], 'implements' => [], 'extends' => [], 'instanceof' => [], 'uses' => []])
+                ->defaultValue([])
```

Comment:
This config was wrong for many years. If you do not have any extension, you should not register any invalid extension config. 

E.g. the `'admins' => []` node tries to register something like this:

```yaml
        sonata_admin:
            extensions:
                admins: []
```

The correct config should be something like this:

```yaml
        sonata_admin:
            extensions:
                app.publish.extension:
                    global: true
                    admins:
                        - app.admin.article
                    implements:
                        - App\Publish\PublishStatusInterface
                    excludes:
                        - app.admin.blog
                        - app.admin.news
                    extends:
                        - App\Document\Blog
                    instanceof:
                        -  App\Document\Page
                    uses:
                        -  App\Trait\Timestampable
```